src/main/java/frc/robot/constants/IntakeConstants.java

```diff
@@ -1,24 +1,32 @@
 package frc.robot.constants;
 
-import edu.wpi.first.math.util.Units;
-
 public class IntakeConstants {
-  public static final int intakeMotorID = 15;
+  public static final int INTAKE_TOP_MOTOR_ID = 15;
+  public static final int INTAKE_LOW_MOTOR_ID = 16; // TODO confirm this is the correct id
+
+  public static final double INTAKE_MOTOR_CURRENT_LIMIT = 5;
 
-  public static final double intakeMotorCurrentLimit = 5;
+  public static final double INTAKE_CORAL_TOP_MOTOR_VOLTAGE = 3;
+  public static final double INTAKE_CORAL_LOW_MOTOR_VOLTAGE = 3;
+  public static final double EJECT_CORAL_TOP_MOTOR_VOLTAGE = -4;
 
-  public static final double intakeVoltage = 3;
-  public static final double ejectVoltage = -4;
+  public static final double INTAKE_ALGAE_LOW_MOTOR_VOLTAGE = -4;
+  public static final double EJECT_ALGAE_LOW_MOTOR_VOLTAGE = 3;
+  public static final double HOLD_ALGAE_LOW_MOTOR_VOLTAGE = -1;
 
   // Used in auto. Teleop is just whileTrue.
-  public static final double intakeTimeout = 2.5;
-  public static final double ejectTimeout = 1.0;
+  public static final double INTAKE_TIMEOUT = 2.5;
+  public static final double EJECT_TIMEOUT = 1.0;
 
-  public static final int CANRANGE_ID = 16;
+  /* Time after we see a coral when intaking to ensure it gets sufficiently intaken */
+  public static final double INTAKE_INSERTION_DELAY = 0.1;
+  /* Time after we stop seeing a coral when ejecting to ensure it gets sufficiently ejected */
+  public static final double EJECT_RELEASE_DELAY = 0.1;
 
-  public static final double canrangeOffset = 0.2721501143002286;
+  /* Distanse where if the can range returns less than it we must have a coral */
+  public static final double HAS_CORAL_DISTANCE = 0.4;
```

Comment:
I believe this is in units of meters, and since we are intaking facing the ground, we'll need to distinguish between the ground (>5 cm away) and having a coral.